test/spec/faros_event_spec.sh

```diff
@@ -357,4 +357,20 @@ Describe 'faros_event.sh'
       The output should equal 'Unrecognized arg(s): Bad_Input Also_Bad Failed.'
     End
   End
+
+  Describe 'Common Tests - '
+    It 'Multiple graphs send multiple events'
+      multi_graph_test() {
+        echo $(
+          ../faros_event.sh CD -k "<api_key>" \
+          --graph "test_1, test_2" \
```

Comment:
when invoked, can one omit `"` specify `--graph test_1,test_2` as well?

Reply:
I actually am going to change it to only support `test_1,test_2` without any whitespace.

Reply:
k